WeightProducer/python/MCSamples_Summer20UL16.py

```diff
@@ -5,6 +5,10 @@
 # 13 TeV miniAOD samples - Summer20UL16
 Summer20UL16samples = [
     # NB: amcatnlo samples have negative weight events, so NumberEvts = # positive - # negative
+    MCSample('TTGJets_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8', '106X_mcRun2_asymptotic_v17-v1', 'RunIISummer20UL16MiniAODv2', 'Const\
```

Comment:
please put each `MCSample` on one line, not split on two lines.

Reply:
I fixed it earlier but I think it was an issue of my editor. I will fix it now